requirements/lint.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with python 3.9
 # To update, run:
 #
-#    pip-compile --allow-unsafe --generate-hashes requirements/lint.in
+#    pip-compile --allow-unsafe --generate-hashes --no-emit-index-url requirements/lint.in
```

Comment:
We never emitted index URLs in the requirements files though so I'm not sure why this is needed. It only works if you define an extra index URL using the `-i` flag, which we don't.

Reply:
if you compile the dependencies and you have `PIP_INDEX_URL` in your env vars pip-compile will add it by default to the generated requirements.txt

Reply:
Ah, right you are, I guess I never had that env var defined when running pip-compile. Sounds good then!